app/src/test/java/protect/card_locker/ImportExportTest.java

```diff
@@ -892,7 +892,7 @@ public void exportImportV2Zip() throws FileNotFoundException {
         int loyaltyCardId = (int) DBHelper.insertLoyaltyCard(mDatabase, "Card 1", "Note 1", new Date(1618053234), new BigDecimal("100"), Currency.getInstance("USD"), "1234", "5432", CatimaBarcode.fromBarcode(BarcodeFormat.QR_CODE), 1, 0, null,0);
         loyaltyCardHashMap.put(loyaltyCardId, DBHelper.getLoyaltyCard(mDatabase, loyaltyCardId));
         DBHelper.insertGroup(mDatabase, "One");
-        List<Group> groups = Arrays.asList(DBHelper.getGroup(mDatabase, "One"));
+        List<Group> groups = Arrays.asList(DBHelper.getGroupByName(mDatabase, "One"));
```

Comment:
Can't comment on the line of the bug, because unchanged, but clearly importing it doing something wrong which is why line 920 fails (after importing, there are 0 cards available)